packages/runtime/container-runtime/garbageCollection.md

```diff
@@ -1,28 +1,38 @@
 # Garbage Collection
-Garbage collection (GC) identifies Fluid objects that are not used and deletes them from the Fluid document. This reduces the size of the Fluid file, the in-memory content and the summary that is uploaded to / downloaded from the server. It also makes processing faster as there is less data to process.
+Garbage collection (GC) is the process by which Fluid Framework performs automatic memory management. GC will run periodically and safely delete objects that are not used. The only responsibility of the users of Fluid Framework is to add and remove references to Fluid objects correctly.
 
-Before understanding the details of how GC works, lets take a look at how to add a reference to Fluid objects when they are in use and remove the reference when they are not in use.
+## Why have Garbage Collection?
+GC reduces the size of the Fluid file, the in-memory content and the summary that is uploaded to / downloaded from the server. It saves COGS on the server and it makes containers load time faster as there is less data to download and process.
 
-## Fluid object references
-- All Fluid objects that are in use must be marked as referenced so that they are not deleted by GC. There are 2 ways to mark objects as referenced:
-  - Create them as `root`. These objects are always referenced and cannot be marked unreferenced later. For example, `root` data stores are always referenced.
+## What do I need to do?
+All Fluid objects that are in use must be marked as referenced so that they are not deleted by GC. Similarly, references to all unused Fluid objects should be removed so that they can be deleted by GC. It is the responsibility of the users of Fluid Framework to correctly add and remove references to Fluid objects.
 
-    `Root` objects can never be deleted so be careful and only create them if they should live forever.
-  - Store a handle ([IFluidHandle](../../../common/lib/core-interfaces/src/handles.ts)) to the object in a referenced DDS that supports handle in its data. For example, a data store's handle can be stored in a referenced `SharedMap` DDS.
-- All references to unused Fluid objects should be removed so that they can be deleted by GC. To remove an object's reference, all its handles should be removed from referenced DDSes.
+## How do I reference / unreference Fluid objects?
+Currently, the only Fluid objects that are eligible for GC are data stores and attachment blobs. The following sections describe how you can mark them as referenced or unreferenced.
+### Data stores
```

Comment:
Let's add a note at the start of this section about DDSes (feel free to adjust the wording, or even split the second part into a separate note later):

> Note: When we speak of a "referenced DDS", we mean a DDS created by a DataStore which is referenced. And likewise, when we speak of storing a handle to a DataStore, storing a handle to any one of its DDSes will mark the whole DataStore as referenced.